Makefile

```diff
@@ -193,9 +193,6 @@ lint-api-schema: build-api-schema
 	$(IN_VENV) codespell -I .ci/ignore-spelling.txt _schema.yaml
 	$(MAKE) remove-api-schema
 
-update-navigation-schema: client-node-deps
```

Comment:
Guessing this was unintentional in the rebase?  Want me to take a pass fixing it up?

Reply:
This seems to have come in when I merged upstream/dev into my fork, but if you can figure it out that would be appreciated!